src/services/CleanupService.ts

```diff
@@ -443,15 +467,25 @@ export const clearSiteDataForThisDomain = async (
 };
 
 export const removeSiteData = async (
-  siteData: string,
-  isChrome: boolean,
+  state: State,
+  siteData: SiteDataType,
+  browserDetect: string,
   domains: string[],
   debug: boolean,
 ): Promise<boolean> => {
-  const listName = isChrome ? 'origins' : 'hostnames';
+  const listName = ((b: string) => {
+    switch (b) {
+      case 'Chrome':
+        return 'origins';
+      case 'Firefox':
+      default:
+        return 'hostnames';
+    }
+  })(browserDetect);
+  const sd = `${siteData[0].toLowerCase()}${siteData.slice(1)}`;
```

Comment:
I think this is common enough to be its own mini function, for the purposes of documentation and easier maintenance in the future.

Reply:
Resolved through 17b7bff